CONTRIBUTING.md

```diff
@@ -3,6 +3,14 @@
 We want to make contributing to this project as easy and transparent as
 possible.
 
+## Build from Source
```

Comment:
Maybe we could just link to the documentation?

- https://swiftpackageindex.com/ml-explore/mlx-swift/main/documentation/mlx/troubleshooting#Installation

I would prefer to have this info in one place in case we need to update it.

Reply:
Yes thanks for review & suggestion. I will update